ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1290,28 +1290,51 @@ def test_sosreport_includes_healthcheck(self, create_logfile):
         )
         assert msg in cmd.stdout_text
 
+    def modify_perms_run_healthcheck(self, filename, modify_permissions,
+                                     expected_permissions):
+        """
+        Modify the ipa logfile permissions and run
+        healthcheck command to check the status.
+        """
+        modify_permissions(self.master, path=filename, mode="0644")
+        returncode, data = run_healthcheck(
+            self.master, "ipahealthcheck.ipa.files", failures_only=True
+        )
+        assert returncode == 1
+        assert len(data) == 1
+        assert data[0]["result"] == "WARNING"
+        assert data[0]["kw"]["path"] == filename
+        assert data[0]["kw"]["type"] == "mode"
+
     def test_ipahealthcheck_verify_perms_for_source_files(self,
-                                                          modify_permissions):
+                                                          modify_permissions,
+                                                          expected_permissions):
```

Comment:
No need to add `expected_permissions` here, it is neither a fixture not a parameter provided to the test.